tests/filecheck/dialects/arith/arith_invalid.mlir

```diff
@@ -1,29 +1,27 @@
 // RUN: xdsl-opt %s --split-input-file --verify-diagnostics | filecheck %s
 
-"builtin.module"() ({
-
-  %lhs, %rhs = "test.op"() : () -> (i32, i64)
-  %res = "arith.addi"(%lhs, %rhs) : (i32, i64) -> i32
-
-  // CHECK: attribute i32 expected from variable 'T', but got i64
-}) : () -> ()
+%lhs, %rhs = "test.op"() : () -> (i32, i64)
+%res = "arith.addi"(%lhs, %rhs) : (i32, i64) -> i32
+// CHECK: attribute i32 expected from variable 'T', but got i64
 
 // -----
 
-"builtin.module"() ({
+%index = "test.op"() : () -> index
+%res = "arith.index_cast"(%index) : (index) -> index
+// CHECK: 'arith.index_cast' op operand type 'index' and result type 'index' are cast incompatible
 
-  %index = "test.op"() : () -> index
-  %res = "arith.index_cast"(%index) : (index) -> index
-  // CHECK: 'arith.index_cast' op operand type 'index' and result type 'index' are cast incompatible
+// -----
 
-}) : () -> ()
+%i32 = "test.op"() : () -> i32
+%res = "arith.index_cast"(%i32) : (i32) -> i32
+// CHECK: 'arith.index_cast' op operand type 'i32' and result type 'i32' are cast incompatible
 
 // -----
 
-"builtin.module"() ({
+%c = arith.constant 1 : si32
```

Comment:
Diff seems a bit confused here, but these are the 2 new invalid test cases.

Reply:
Maybe tiny PR to dedent first I'll insta approve?